requirements.txt

```diff
@@ -5,3 +5,5 @@ Theano==0.8
 joblib==0.9.3
 scikit-learn==0.17
 tabulate==0.7.5
+graphviz
+pydot
```

Comment:
The `requirements.txt` currently excludes requirements for visualization.  So safe to remove these two for now.  They'll still be installed via the `visualization_require` list defined in `setup.py`.